lib/ssh_pearl.sh

```diff
@@ -34,16 +34,19 @@ function ssh_pearl(){
 
     local rcScript=$(_aggregate_scripts "$PEARL_HOME/sshrc" "$PEARL_HOME/sshrc.d" | gzip | base64)
     local inputrcScript=$(_aggregate_scripts "$PEARL_HOME/sshinputrc" "$PEARL_HOME/sshinputrc.d" | gzip | base64)
+    local remoteTmpDirs=( "/tmp" "\$HOME" )
 
     CMD="
+        for tmpDir in ${remoteTmpDirs[*]}; do [[ -w \"\$tmpDir\" ]] && { foundTmpDir=\"\$tmpDir\"; break; } done
+        [[ -z \"\$foundTmpDir\" ]] && { echo -e \"couldn't find writable tempdirs on the server. Aborting.\"; exit 1; };
```

Comment:
Similarly to the other comments I would redirect it to stderr:

    [[ -z \"\$foundTmpDir\" ]] && { echo >&2 \"Could not find writable tempdirs on the server. Aborting.\"; exit 1; };